apiserver/modelmanager/modelmanager.go

```diff
@@ -326,6 +326,11 @@ func (m *ModelManagerAPI) CreateModel(args params.ModelCreateArgs) (params.Model
 	}
 	defer st.Close()
 
+	err = st.ReloadSpaces(env)
+	if err != nil {
```

Comment:
I often like to do these as:
if err := st.ReloadSpaces(env); err != nil {
}

it means the scope of the 'err' variable is only for the if/else blocks.

Not something that needs to be changed, as we're certainly not consistent everywhere. Just something I often do.